src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    public Figure getRandomFigure(int maxSize) {
```

Comment:
Consider making the `maxSize` parameter a constant if it doesn't change, or providing it through the constructor if it's meant to be configurable per instance.